src/components/Main.js

```diff
@@ -2,17 +2,164 @@ import { Component } from 'react';
+          "title": "Addax",
+          "description": "This guy is basically extinct but survives well in captivity, so they're frequently found in zoos",
+          "keyword": "addax",
+          "horns": 2
+        },
+        {
+          "image_url": "https://cbsnews3.cbsistatic.com/hub/i/r/2013/03/05/5b414225-a645-11e2-a3f0-029118418759/thumbnail/620x350/2d4cf24685b45c22912e64d2004fec8d/Baby_Mouflon_Wild_Sheep.jpg",
+          "title": "Baby mouflon",
+          "description": "The cuteness that is a baby mouflon asleep",
+          "keyword": "mouflon",
+          "horns": 2
+        },
+        {
+          "image_url": "https://images.unsplash.com/photo-1514036783265-fba9577fc473?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1350&q=80",
+          "title": "Happy Jackson's Chameleon",
+          "description": "These are really common in Hawaii",
+          "keyword": "chameleon",
+          "horns": 2
+        },
+        {
+          "image_url": "https://imgc.allpostersimages.com/img/print/posters/dlillc-jackson-s-chameleon_a-G-13448768-14258384.jpg",
+          "title": "Serious Jackson's Chameleon",
+          "description": "This one is very serious.",
+          "keyword": "chameleon",
+          "horns": 3
+        },
+        {
+          "image_url": "https://www.nps.gov/band/learn/nature/images/short-horned-lizard-open-mouth-18.jpg?maxwidth=650&autorotate=false",
+          "title": "Horned Lizard",
+          "description": "Fave food: ants",
+          "keyword": "lizard",
+          "horns": 100
+        },
+        {
+          "image_url": "https://upload.wikimedia.org/wikipedia/commons/thumb/d/df/Smaug_par_David_Demaret.jpg/290px-Smaug_par_David_Demaret.jpg",
+          "title": "Smaug",
+          "description": "Fan illustration of Smaug from 'The Hobbit'",
+          "keyword": "dragon",
+          "horns": 100
+        }
+      ]
+    };
+  }
   render() {
+    console.log(this.state.hornedBeasts);
     return (
-      <div>
-        <HornedBeast title='UniWhal' imageURL='http://3.bp.blogspot.com/_DBYF1AdFaHw/TE-f0cDQ24I/AAAAAAAACZg/l-FdTZ6M7z8/s1600/Unicorn_and_Narwhal_by_dinglehopper.jpg' description='A unicorn and a narwhal nuzzling their horns' />
-
-        <HornedBeast title='Rhino Family' imageURL='https://images.unsplash.com/photo-1512636618879-bbe79107e9e3?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=bd9460ee6d1ddbb6b1ca7be86dfc4590&auto=format&fit=crop&w=1825&q=80' description='Parent rhino with two babies' />
-
-        <HornedBeast title='Unicorn Head' imageURL='https://www.dhresource.com/0x0s/f2-albu-g5-M00-1A-11-rBVaI1hsIIiALxKzAAIHjSU3VkE490.jpg/wholesale-halloween-costume-prop-unicorn.jpg' description='Someone wearing a very silly unicorn head mask' />
-      </div>
+      <>
+        {this.state.hornedBeasts.map(beast => {
+          return <HornedBeast title={beast.title} imageURL={beast.image_url} description={beast.description} />;
+        })}
+      </>
```

Comment:
Feedback here: Add key prop in, anytime we map() we will need to add the key. When running the app, if you right click to open chrome dev tools, and check console you will see error for this. 👍